src/main/webapp/app/exam/manage/student-exams/student-exams.component.ts

```diff
@@ -82,7 +81,7 @@ export class StudentExamsComponent implements OnInit {
             const examObservable = this.examManagementService.find(this.courseId, this.examId, true).pipe(
```

Comment:
Just adding it here for line 75/74 since I can't mark it directly.
Looks like you forgot a `null` there.